docker-stack.yml

```diff
@@ -0,0 +1,83 @@
+version: '3.3'
+
+networks:
+  net:
```

Comment:
Just wanted to rename the network to `elk` to stay consistent with our compose deployment.